### PR TITLE
added invalidate cache button in project admin page

### PR DESCRIPTION
**Pull request by FabriceIRANKUNDA**
## Summary

_A summary of changes being made in this PR_
Admin can invalidate the storage provider's cache

## Checklist
- [ ] Pre-push checks pass `npm run check`
  - [ ] Linter passing via `npm run lint`
  - [ ] Unit & Integration tests passing via `npm run test:packages`
  - [ ] Docker build process passing via `npm run build-client`
- [ ] If this PR is still a WIP, convert to a draft 
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References
#4153 

closes https://github.com/XRFoundation/XREngine/issues/4153

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

File: packages/server-core/src/projects/project/project.service.ts [hunk 1]
```diff
@@ -36,6 +37,14 @@ export default (app: Application): void => {
     }
   })
 
+  app.use('project-invalidate', {
+    patch: async ({ projectName }, params) => {
+      if (projectName) {
+        return await useStorageProvider().createInvalidation([`projects/${projectName}*`])
```
Comment (HexaField) [inline packages/server-core/src/projects/project/project.service.ts:43]: we need a `/` between `{projectName}` and `*`

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/server-core/src/projects/project/project.service.ts (hunk 1 above)
Comment (HexaField) [reply packages/server-core/src/projects/project/project.service.ts:43]: nevermind, i am wrong! https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Invalidation.html#invalidation-specifying-objects

**Review by HexaField — APPROVED** [review]

File: packages/server-core/src/projects/project/project.service.ts [hunk 2]
```diff
@@ -36,6 +37,14 @@ export default (app: Application): void => {
...
+      }
+    }
+  })
+
   app.service('project-build').hooks({
```
Comment (barankyle) [inline packages/server-core/src/projects/project/project.service.ts:48]: There should be a .hooks for the new project-invalidate service. It should also add authenticate() and restrictUserRole('admin') on the patch action.

Comment (HexaField) [reply packages/server-core/src/projects/project/project.service.ts:48]: yes, really we should move all these to be methods inside of the 'project' patch action. i made a bad pattern here. let's recitfy it here.

**Review by HexaField — APPROVED** [review]

**Pull request merged**